Editor/TestTools/EditorImguiTester/Windows/EditorImguiTestsExplorer.cs

```diff
@@ -268,7 +263,7 @@ private void DrawTestView(IEditorImguiTest test)
             EditorGUILayout.EndHorizontal();
         }
 
-        private void DrawTestStateIndicator(Vector2 position, TestState state)
+        private static void DrawTestStateIndicator(Vector2 position, TestState state)
```

Comment:
Why? 🤔

Reply:
It is a private helper method that does not use anything from the calling instance. No reason to keep it instance method.